File: Examples/Ruby/Charts/delete_worksheet_delete_chart.rb
require 'aspose_cells_cloud'

class Chart

  include AsposeCellsCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @cells_api = CellsApi.new  
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def delete_worksheet_delete_chart
    file_name = "Sample_Test_Book.xls"
    upload_file(file_name)

    sheet_name = "Sheet5"
    chart_index = 0
    response = @cells_api.delete_worksheet_delete_chart(file_name, sheet_name, chart_index)
  end

end

chart = Chart.new()
puts chart.delete_worksheet_delete_chart

File: Examples/PHP/Charts/DeleteWorksheetDeleteChart.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Cells\CellsApi;
use Aspose\Cells\AsposeApp;

class Chart {

	public $cells;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->cells = new CellsApi();
	}

	public function deleteWorksheetDeleteChart() {
    	// Upload file to Aspose Cloud Storage
		$fileName = "Sample_Test_Book.xls";
		Utils::uploadFile($fileName);

        $result = $this->cells->DeleteWorksheetDeleteChart($name=$fileName, $sheetName="Sheet5", $chartIndex="0", $storage = null, $folder = null);
        print_r($result);
	}
}

$chart = new Chart();
$chart->deleteWorksheetDeleteChart();

?>
//ExEnd:
